server/bin/pbench-audit-server.sh

```diff
@@ -727,9 +727,9 @@ if [[ $? -ne 0 ]]; then
 fi
 eTS=$(timestamp)
 if [[ -s ${archive_report} ]]; then
-    printf "\nstart-${sTS}: archive hierarchy: $ARCHIVE\n" | tee -a ${report}
+    log_info "\nstart-${sTS}: archive hierarchy: $ARCHIVE" "${report}"
```

Comment:
Are we sure we want to do this kind of change for the audit server code?

This means that new-line characters will get into the log statements.  If we could capture the message without the new-line formatting characters, log that, and then add it to the report with the new-line characters that would be ideal.

However, that may not be worth doing.

Could we consider removing the use of the `tee` command, and just having the `printf` and `cat` commands output to the `${report}`, and then add log_info statements to before the various `$(timestamp)` calls?

Reply:
ok

Reply:
@portante according to your suggestion I tried printf with cat but it doesn't seems to work. The approaches that I used are these that seems to work a little, other are way apart but each leads to change in gold files:
printf "message" > ${report} | cat ${report}
printf "message" > ${report} | log_info | cat ${report}

Reply:
II would be something like this:

     printf  "message" >>  ${report}
     cat ${archive_report}  >> ${report}
     log_info "some other message"

Basically, the detailed information ends up in the report; the log only has a brief indication of what happened (in a single line, so that it's easier to scan).

Reply:
@portante:  It seems that it's not quite clear yet what we should do with audit, so it might make sense to leave it out of this PR and consider it separately - would it make sense to leave the audit code unchanged in this PR?

Reply:
@ndokos, that seems like a reasonable approach.

Reply:
@ndokos the suggestion that you provided I tried that as well earlier but that is leading to the change in gold file as well.

Ok. I will leave it unchanged